### PR TITLE
Use RedirectSlashes middleware by default Goa mux 

### PR DESCRIPTION
**Pull request by tchssk**
https://github.com/goadesign/goa/pull/3364#issuecomment-1734770313

### CODE REVIEW

**Comment by tchssk** [comment]
DeepSource: Test coverage looks always timed out.

File: http/mux.go [hunk 1]
```diff
@@ -85,6 +86,7 @@ func NewMuxer() ResolverMuxer {
 		w.WriteHeader(http.StatusNotFound)
 		enc.Encode(NewErrorResponse(ctx, fmt.Errorf("404 page not found"))) // nolint:errcheck
 	}))
+	r.Use(middleware.RedirectSlashes)
```
Comment (haruyama) [inline http/mux.go]: https://github.com/go-chi/chi/blob/1129e362d6cce6e3805e3bc8dfbaeb34b5129789/middleware/strip_test.go#L105-L107

Comment (tchssk) [reply http/mux.go]: Thank you. Fixed. 3fbf066e93645b3640f49a1abd29c4cc8a1299fd

**Comment by raphael** [comment]
This is great, thank you! (I need to fix the build failing for PRs coming from forks)

**Pull request merged**